### ISSUE TITLE
Helpers for an actions share callback queues

### ISSUE THREAD

**Issue by duncanfwalker** [lexich/redux-api#134]
When using the crud set of action helpers the callback queue is shared. 

In the following example the alert would never appear:

```javascript
const thenDoAGet = () => dispatch(restApi.actions.test.get({id: '1', undefined, () => alert("never gets called") }))
dispatch(restApi.actions.test.put({id: '1' }, {body: {}}, thenDoAGet))
```

This is happened because a new PubSub is created for the 'test' **action** (see https://github.com/lexich/redux-api/blob/master/src/actionFn.js#L25), rather than each helper. This means that when the helper resolves it clears the pubsub, removing the second callback.

Instead of this behaviour I would expect the callbacks for each **helper** to be independent from one another.  

The use case is redirecting to a new React route after a PUT, and the new route making a GET on the same entity

**Comment by lexich** [comment]
I wrote test https://github.com/lexich/redux-api/blob/master/test/redux_spec.js#L513-L538 and all ok. Can you explain your case with test?

**Comment by duncanfwalker** [comment]
The second action gets triggered ok, but not its callback. In the test below the spy never gets called so it doesn't resolve.

For actionFn_spec.js

```javascript
  it("chained callbacks all resolve",  function() {
    const meta = {
      transformer,
      crud: true,
      fetch(url, opts) {
        return new Promise(resolve => resolve({ url, opts }))
      }
    };
    const api = actionFn("/test/:id", "test", null, ACTIONS, meta);

    let callCount = 0;
    let spy = (resolve) => {
      callCount++;
      resolve();
    };
    let chainedAction = (resolve) => api.get({id: 1}, (resolve) => spy(resolve));

    return new Promise((resolve)=> {
      return api.get({ id: 1 }, () => chainedAction(resolve))(function () {}, getState);
    })
      .then(()=> expect(callCount).to.have.length.equal(1));
  });
```

**Comment by lexich** [comment]
@duncanfwalker I fix your issue in 0.10.3 Thanks for your feedback. Pls check it. 👍

**Comment by duncanfwalker** [comment]
Fantastics - all works as expected. Thanks for the quick response/fix
